docs-ref-services/latest/servicebus.md

```diff
@@ -15,402 +15,39 @@ ms.service: service-bus
-sbs.send_topic_message('taskdiscussion', msg)
-```
-
-The **send\_topic\_message_batch** method can be used to send  several messages at once:
-
-```python
-from azure.servicebus.control_client import Message
-
-msg1 = Message(b'Hello World!')
-msg2 = Message(b'Hello World again!')
-sbs.send_topic_message_batch('taskdiscussion', [msg1, msg2])
-```
-
-Please consider that in Python 3 a str message will be utf-8 encoded and you should have to manage your encoding yourself in Python 2.
-
-A client can then create a subscription and start consuming messages by calling the **create\_subscription** method followed by the **receive\_subscription\_message** method. Please note that any messages sent before the subscription is created will not be received.
-
-```python
-from azure.servicebus.control_client import Message
-
-sbs.create_subscription('taskdiscussion', 'client1')
-msg = Message('Hello World!')
-sbs.send_topic_message('taskdiscussion', msg)
-msg = sbs.receive_subscription_message('taskdiscussion', 'client1')
-```
-
-### Event Hub
-
-Event Hubs enable the collection of event streams at high throughput, from a diverse set of devices and services.
+## Libraries for data access
 
-The **create\_event\_hub** method can be used to create an event hub:
+The latest version of the Azure Servicebus libraries is version 7.x.x. We highly recommend using version 7.x.x for new applications.
 
-```python
-sbs.create_event_hub('myhub')
-```
+To update existing applications to version 7.x.x, please follow the [migration guide](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/servicebus/azure-servicebus/migration_guide.md).
 
-To send an event:
+### Version 7.x.x
 
-```python
-sbs.send_event('myhub', '{ "DeviceId":"dev-01", "Temperature":"37.0" }')
-```
+The version 7.x.x client library for Python is part of the Azure SDK for Python. The source code for the Azure ServiceBus client libraries for Python is available on [GitHub](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/servicebus).
 
-The event content is the event message or JSON-encoded string that contains multiple messages.
+Use the following library to manage your Azure Service Bus resources like namespaces, queues, topics, subscriptions and rules via the Azure Resource Manager, you would use the below package:
 
-### Advanced features
+| Library | Reference | Package | Source |
+|----------------------------------------|-------------------------------------------------------------|-----------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------|
+|    azure-servicebus   |    [Reference](https://docs.microsoft.com/python/api/overview/azure/servicebus-readme?view=azure-python)    |    [PyPI](https://pypi.org/project/azure-servicebus/)    |    [GitHub](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/servicebus/azure-servicebus)    |
```

Comment:
Any reason to have "Library" and "Package" as separate columns? Can we not make `azure-servicebus` as the link text and add the pypi link to it?

Instead, we can have a column for "Samples"